packages/rx_bloc_cli/mason_templates/bricks/feature_profile/__brick__/lib/feature_profile/views/profile_page.dart

```diff
@@ -33,212 +33,205 @@ class ProfilePage extends StatefulWidget {
 }
 
 class _ProfilePageState extends State<ProfilePage> {
-  @override
-  void initState() {
-     context.read<ProfileBlocType>().events.loadNotificationsSettings(); {{#enable_pin_code}}
```

Comment:
@JvnSlv looking at the BLoC it seems that we have added an initial trigger `startWith(null)`, which probably makes the invocation of the `loadNotificationsSettings()` redundant. Please confirm this change won't have negative impact.

Reply:
You're right; `startWith(null)` makes `loadNotificationsSettings()` redundant, and removing it won't have any negative impact.